posydon/unit_tests/grids/test_MODELS.py

```diff
@@ -0,0 +1,39 @@
+"""Unit tests of posydon/grids/MODELS.py
+"""
+
+__authors__ = [
+    "Matthias Kruckow <Matthias.Kruckow@unige.ch>"
+]
+
+# import the module which will be tested
+import posydon.grids.MODELS as totest
+
+# import other needed code for the tests, which is not already imported in the
+# module you like to test
+
+
+# define test classes collecting several test functions
+class TestElements:
+    # check for objects, which should be an element of the tested module
+    def test_dir(self):
+        elements = ['MODELS', 'NEUTRINO_MASS_LOSS_UPPER_LIMIT',\
+                    'STATE_NS_STARMASS_UPPER_LIMIT', '__authors__',\
+                    '__builtins__', '__cached__', '__doc__', '__file__',\
+                    '__loader__', '__name__', '__package__', '__spec__']
+        assert dir(totest) == elements, "There might be added or removed "\
+                                        +"objects without an update on the "\
+                                        +"unit test."
```

Comment:
```suggestion
        missing_in_test = list(set(dir(totest)) - set(elements))
        missing_in_module = list(set(elements) - set(dir(totest)))
        
        assert len(missing_in_test) == 0, \
            "The unit test could not find the following "\
            +f"objects in {totest.__name__}: {missing_in_test}. "\
            +f"\nThese objects may have been removed from {totest.__name__}. "\
            +"Consider updating this unit test by removing these objects from "\
            +"its elements list."
        
        assert len(missing_in_module) == 0, \
           f"The module {totest.__name__} has additional "\
           +"objects not considered by this unit test. They are: "\
           +f"{missing_in_test}.\nThese objects may have been "\
           +f"added to {totest.__name__}. Consider updating this unit "\
           +"test by adding these objects to its elements list."
```

I suggest something like this to help make debugging a little easier, here and similarly for the other unit tests.